addons/mail/static/src/core/web/chatter.js

```diff
@@ -131,7 +131,10 @@ export class Chatter extends Component {
                             return;
                         }
                     }
-                    files.forEach((file) => this.attachmentUploader.uploadFile(file));
+                    await Promise.all(files.map((file) => this.attachmentUploader.uploadFile(file)))
```

Comment:
Note there is a slight difference here where the `isAttachmentBoxOpened` will be done after all promise resolve, whereas it was immediately done before.

Maybe we'd want to move the new code in a then instead of await, or move the line above?

Possibly opening the box is useful to see progress bars.

Reply:
Good catch, I moved that line above